string axis    #one of surge, sway, heave, roll, pitch, yaw
float64 value
---
bool success   #whether or not update succeeded
